js/data/locale/ky/sysres.json

```diff
@@ -187,48 +187,48 @@
-    "1#in 1 week|#in {num} weeks": "one#‏{num} аптадан кийин|#‏{num} аптадан кийин",
-    "1#in 1 month|#in {num} months": "one#‏{num} айдан кийин|#‏{num} айдан кийин",
-    "1#in 1 year|#in {num} years": "one#‏{num} жылдан кийин|#‏{num} жылдан кийин",
-    "1#in 1 sec|#in {num} sec": "one#‏{num} секунддан кийин|#‏{num} секунддан кийин",
-    "1#in 1 min|#in {num} min": "one#‏{num} мүнөттөн кийин|#‏{num} мүнөттөн кийин",
-    "1#in 1 hr|#in {num} hrs": "one#‏{num} сааттан кийин|#‏{num} сааттан кийин",
-    "1#in 1 dy|#in {num} dys": "one#‏{num} күндөн кийин|#‏{num} күндөн кийин",
-    "1#in 1 wk|#in {num} wks": "one#‏{num} аптадан кийин|#‏{num} аптадан кийин",
-    "1#in 1 mon|#in {num} mons": "one#‏{num} айдан кийин|#‏{num} айдан кийин",
-    "1#in 1 yr|#in {num} yrs": "one#‏{num} жылдан кийин|#‏{num} жылдан кийин",
-    "#in {num}s": "one#‏{num} сек. кийин|#‏{num} сек. кийин",
-    "#in {num}mi": "one#‏{num} мүн. кийин|#‏{num} мүн. кийин",
-    "#in {num}h": "one#‏{num} саат. кийин|#‏{num} саат. кийин",
-    "#in {num}d": "one#‏{num} күн. кийин|#‏{num} күн. кийин",
-    "#in {num}w": "one#‏{num} апт. кийин|#‏{num} апт. кийин",
-    "#in {num}mo": "one#‏{num} айд. кийин|#‏{num} айд. кийин",
-    "#in {num}y": "one#‏{num} жыл. кийин|#‏{num} жыл. кийин",
-    "1#1 second ago|#{num} seconds ago": "one#‏{num} секунд мурун|#‏{num} секунд мурун",
-    "1#1 minute ago|#{num} minutes ago": "one#‏{num} мүнөт мурун|#‏{num} мүнөт мурун",
-    "1#1 hour ago|#{num} hours ago": "one#‏{num} саат мурун|#‏{num} саат мурун",
-    "1#1 day ago|#{num} days ago": "one#‏{num} күн мурун|#‏{num} күн мурун",
-    "1#1 week ago|#{num} weeks ago": "one#‏{num} апта мурун|#‏{num} апта мурун",
-    "1#1 month ago|#{num} months ago": "one#‏{num} ай мурун|#‏{num} ай мурун",
-    "1#1 year ago|#{num} years ago": "one#‏{num} жыл мурун|#‏{num} жыл мурун",
-    "1#1 sec ago|#{num} sec ago": "one#‏{num} секунд мурун|#‏{num} секунд мурун",
-    "1#1 min ago|#{num} min ago": "one#‏{num} мүнөт мурун|#‏{num} мүнөт мурун",
-    "1#1 hr ago|#{num} hrs ago": "one#‏{num} саат мурун|#‏{num} саат мурун",
-    "1#1 dy ago|#{num} dys ago": "one#‏{num} күн мурун|#‏{num} күн мурун",
-    "1#1 wk ago|#{num} wks ago": "one#‏{num} апта мурун|#‏{num} апта мурун",
-    "1#1 mon ago|#{num} mons ago": "one#‏{num} ай мурун|#‏{num} ай мурун",
-    "1#1 yr ago|#{num} yrs ago": "one#‏{num} жыл мурун|#‏{num} жыл мурун",
-    "#{num}s ago": "one#‏{num} сек. мурун|#‏{num} сек. мурун",
-    "#{num}mi ago": "one#‏{num} мүн. мурун|#‏{num} мүн. мурун",
-    "#{num}h ago": "one#‏{num} саат. мурун|#‏{num} саат. мурун",
-    "#{num}d ago": "one#‏{num} күн мурун|#‏{num} күн мурун",
-    "#{num}w ago": "one#‏{num} апт. мурун|#‏{num} апт. мурун",
-    "#{num}mo ago": "one#‏{num} ай мурун|#‏{num} ай мурун",
-    "#{num}y ago": "one#‏{num} жыл мурун|#‏{num} жыл мурун",
+    "1#in 1 second|#in {num} seconds": "one#{num} секунддан кийин|#{num} секунддан кийин",
```

Comment:
Here either... what's the difference? Why are these files showing up here?

Reply:
The difference is `200f` rtl mark which is removed. When the tool generating data, if it's rtl, it added `200f` in front which is removed now in this data